### PR TITLE
Create composer.json for instalation with composer

### PR DESCRIPTION
**Pull request by alphp**
The installation is much simpler with Composer and https://github.com/RobLoach/component-installer

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    }
+  ],
+  "require": {
+    "robloach/component-installer": "*",
+    "components/jquery": "*"
+  },
+  "extra": {
+    "component": {
+      "scripts": [
+        "jquery.timepicker.js"
```
Comment (jonthornton) [inline composer.json]: Should this be the minified script?

Comment (alphp) [reply composer.json]: Json only enables composer installation, the user will use jquery.timepicker.js or jquery.timepicker.min.js as appropriate.
That is why the definition is given without minimizing the script.

**Comment by jonthornton** [comment]
Thanks for submitting this! I've merged it with a few small changes.

**Comment by alphp** [comment]
You can test it:
```
composer config repositories.jonthornton-jquery-timepicker vcs "https://github.com/jonthornton/jquery-timepicker"
composer require jonthornton/jquery-timepicker:dev-master
```
Composer creates a "components" folder that contains the "jquery-timepicker" whith all files spec in "files" of composer.json.

**Comment by alphp** [comment]
You can then register the "package" at https://packagist.org

**Pull request merged**